src/notifications/builder.ts

```diff
@@ -267,6 +278,19 @@ export class NotificationBuilder {
     return this;
   }
 
+  avatarTopReaderBadge(
+    ctx: NotificationUserTopReaderContext,
+  ): NotificationBuilder {
+    this.avatars.push({
+      type: 'top_reader_badge',
+      name: ctx.keyword.flags.title,
+      targetUrl: '',
```

Comment:
This needs something like notification?topreader&badgeId no?

Reply:
I don't use that URL in frontend, but it has to be set in DB, se empty string it is 😬